04-ruby-mvc-sinatra/active-record-intro/readme.md

```diff
@@ -158,6 +163,9 @@ gem "pry"  # this gem allows access to REPL
 Then I'm going to run `$ bundle install` in the terminal.
 
 ### Setup Ruby - Tunr (You Do - 10 / 70)
+If you did not complete the initial SQL setup for Tunr above:
```

Comment:
I think this is correct, but I'm gonna ping @jshawl for clarification. For some reason I always have trouble in this scenario:

1. Students clone an exercise
2. I push a new branch after they've cloned
3. They need to checkout that branch.